### PR TITLE
DOC-309 Add landing page for Redpanda Connect

### PR DESCRIPTION
**Pull request by JakeSCahill**
## Description

Review deadline: 13 December

## Page previews

https://deploy-preview-131--redpanda-connect.netlify.app/redpanda-connect/home/

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by Deflaimun** [comment]
The intro feels like a wall of text. We should shorten it a bit 
image

File: antora.yml [hunk 1]
```diff
@@ -19,6 +19,42 @@ asciidoc:
     # This is replaced by the version fetcher extension
     # https://github.com/redpanda-data/docs-extensions-and-macros/blob/main/README.adoc#version-fetcher
     latest-connect-version: 4.30.1
+    # Data for the home page
+    page-home-intro: Redpanda Connect is a declarative data streaming service that solves a wide range of data engineering problems with simple, chained, stateless processing steps. Effortlessly integrate data across systems with hundreds of prebuilt connectors, change data capture (CDC) capabilities, and YAML-configurable workflows. Redpanda Connect empowers you to build scalable, high-performance data pipelines with minimal complexity.
```
Comment (Deflaimun) [inline antora.yml:23]:
```suggestion
    page-home-intro: Redpanda Connect is a declarative data streaming that simplifies complex data engineering problems with simple, chained, stateless processing steps. Integrate data across systems with hundreds of prebuilt connectors, change data capture (CDC) capabilities, and YAML-configurable workflows. 
```

Comment (Deflaimun) [reply antora.yml:23]: an effort to reduce the size, but this one is quite challenging

**Comment by JakeSCahill** [comment]
> The intro feels like a wall of text. We should shorten it a bit image

Thanks! I've made all descriptions more concise. Please take another look 😄

File: antora.yml [hunk 2]
```diff
@@ -19,6 +19,42 @@ asciidoc:
     # This is replaced by the version fetcher extension
     # https://github.com/redpanda-data/docs-extensions-and-macros/blob/main/README.adoc#version-fetcher
     latest-connect-version: 4.30.1
+    # Data for the home page
+    page-home-intro: Redpanda Connect is a data streaming service for building scalable, high-performance data pipelines that drive real-time analytics and actionable business insights. Effortlessly integrate data across systems with hundreds of prebuilt connectors, change data capture (CDC) capabilities, and YAML-configurable pipelines.
```
Comment (asimms41) [inline antora.yml:23]:
```suggestion
    page-home-intro: Redpanda Connect is a data streaming service for building scalable, high-performance data pipelines that drive real-time analytics and actionable business insights. Integrate data across systems with hundreds of prebuilt connectors, change data capture (CDC) capabilities, and YAML-configurable pipelines.
```

**Review by asimms41 — APPROVED** [review]
Looks fab!! Thanks Jake.

Just one minor comment. Completely agree about having the connector catalog in just one place.

**Review by Deflaimun — APPROVED** [review]
Looks really good now!

**Pull request merged**